virt/nodes/virt-managing-node-labeling-obsolete-cpu-models.adoc

```diff
@@ -8,8 +8,4 @@ toc::[]
 
 You can schedule a virtual machine (VM) on a node as long as the VM CPU model and policy are supported by the node.
 
-include::modules/virt-about-node-labeling-obsolete-cpu-models.adoc[leveloffset=+1]
```

Comment:
@SNiemann15, :slightly_smiling_face:, hi, I think that if you want to remove these two included sections, then you probably also need to remove these two module files:
```
modules/virt-about-node-labeling-obsolete-cpu-models.adoc
modules/virt-about-node-labeling-cpu-features.adoc
```
If you remove these files, I suggest running a search (`$ grep -rine "<filename>"`) to ensure that these modules are not included in any other assembly file.

Reply:
@max-cx Not sure if I can remove them? What happens if there are changes to those modules in the versions where this was supported? 

fyi: @ousleyp

Reply:
Moved to unused_topics